keps/sig-autoscaling/1610-container-resource-autoscaling/README.md

```diff
@@ -281,19 +439,594 @@ unit tests will be added:
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `HPAContainerMetrics`
+  - Components depending on the feature gate: `kube-apiserver`, `kube-controller-manager`
+
+###### Does enabling the feature change any default behavior?
 
-### Upgrade/Downgrade Strategy
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
 
-For cluster upgrades the HPAs from the previous version will continue working as before. There
-is no change in behavior or flags which have to be enabled or disabled.
+No.
 
-A new feature gate `HPAContainerMetrics` is added in version 1.20 which is turned off by default.
-The new feature can only be used when this feature gate is enabled.
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
+
+Feature gates are typically disabled by setting the flag to `false` and
+restarting the component. No other changes should be necessary to disable the
+feature.
+
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+The feature can be disabled in Alpha and Beta versions
+by restarting kube-apiserver and kube-controller-manager with the feature-gate off.
+In terms of Stable versions, users can choose to opt-out by not setting the
+`ContainerResource` type metric in their HPA.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+HPA with `ContainerResource` type metric can be created and can be handled by HPA controller.
```

Comment:
Related to question above, if HPA w/ `ContainerResource` exist from when the feature was previously enabled, workloads could potentially undergo scale operations? Can we clarify that here as well.